tests/subsys/debug/thread_analyzer/testcase.yaml

```diff
@@ -47,6 +47,16 @@ tests:
       - CONFIG_THREAD_ANALYZER_USE_LOG=y
       - CONFIG_LOG=y
       - CONFIG_LOG_MODE_IMMEDIATE=y
+    platform_exclude:
+      # The following platforms forcibly enable CONFIG_LOG_PRINTK which is
```

Comment:
where is CONFIG_LOG_PRINTK being enabled for those platforms?

Reply:
Due to it being selected by https://github.com/zephyrproject-rtos/zephyr/blob/dfcc3b6db4005786ea4228a2fd672794222202e2/subsys/logging/backends/Kconfig.rtt#L104-L107

Reply:
can we just do platform_allow: qemux_86 instead of blacklisting random boards, this traditionally becomes a whack a mole thing where we end up excluding boards. Lets just test userspace with qemu_x86 and be done with it :)

Reply:
I've decided to use `platform_allow` to limit the scope, so that I don't have to keep nudging the platform filter lists.